blips/blip-0001.mediawiki

```diff
@@ -0,0 +1,242 @@
+
+* Replaced or Obsolete - when a Final bLIP is no longer relevant, its status may be changed to Replaced or Obsolete (which is equivalent to Replaced). This change must also be objectively verifiable and/or discussed.
+
+====Auxiliary Files====
+
+bLIPs may include auxiliary files such as diagrams. Auxiliary files should be included in a subdirectory for that bLIP, or must be named bLIP-XXXX-Y.ext, where "XXXX" is the bLIP number, "Y" is a serial number (starting at 1), and "ext" is replaced by the actual file extension (e.g. "png").
+
+==Licensing==
+
+New bLIPs may be accepted with the following licenses. Each new bLIP must identify at least one acceptable license in its preamble. The License header in the preamble must be placed after the Created header. Each license must be referenced by their respective abbreviation given below.
+
+For example, a preamble might include the following License header:
+ 
+   License: BSD-2-Clause
+             GNU-All-Permissive
+
+In this case, the bLIP text is fully licensed under both the OSI-approved BSD 2-clause license as well as the GNU All-Permissive License, and anyone may modify and redistribute the text provided they comply with the terms of *either* license. In other words, the license list is an "OR choice", not an "AND also" requirement.
+
+It is also possible to license source code differently from the bLIP text. An optional License-Code header is placed after the License header. Again, each license must be referenced by their respective abbreviation given below.
+
+For example, a preamble specifying the optional License-Code header might look like:
+ 
+   License: BSD-2-Clause
+             GNU-All-Permissive
+   License-Code: GPL-2.0+
+
+In this case, the code in the bLIP is not available under the BSD or All-Permissive licenses, but only under the terms of the GNU General Public License (GPL), version 2 or newer. If the code were to be available under *only* version 2 exactly, the "+" symbol should be removed from the license abbreviation. For a later version (eg, GPL 3.0), you would increase the version number (and retain or remove the "+" depending on intent).
+ 
+   License-Code: GPL-2.0   # This refers to GPL v2.0 *only*, no later license versions are acceptable.
```

Comment:
Do we have a legal opinion saying that GPL/BSD/MIT makes sense for text? I figure'd we'd go CC-0 or so. Also I think we can be way more restrictive here, why not just require pubdom/CC-0? Otherwise we get lots of useless debate around licenses.

Reply:
No not at all, this is all just copied from BIP-0002 as an assumed best practice. If pubdom/CC-0 are sufficient, I have no issue with condensing what's otherwise a long and convoluted section.

Line 238 does say Public Domain is no longer acceptable because "In some jurisdictions, public domain is not recognised as a legitimate legal action, leaving the bLIP simply copyrighted with no redistribution or modification allowed at all." ...is that the case or can it be ignored?